src/app/reactUtils.ts

```diff
@@ -18,6 +18,45 @@ export const useDrawFocus = <E extends HTMLElement = HTMLElement>() => {
   return ref;
 };
 
+// Exported to allow testing
+export function createTrapTab(ref: React.MutableRefObject<HTMLElement | null>) {
+  return (event: KeyboardEvent) => {
+    if (event.key !== 'Tab') { return; }
+
+    const el = ref.current;
+    if (!el) { return; }
+
+    const focusableItemQuery =
+      'button, [href], input, select, textarea, [tabindex]:not([tabindex="-1"])';
```

Comment:
Should we skip `disabled` attributes here? I'm thinking disabled-but-visible inputs could be a concern.